src/univariate/continuous/lognormal.jl

```diff
@@ -89,6 +89,12 @@ function entropy(d::LogNormal)
     (1 + log(twoπ * σ^2))/2 + μ
 end
 
+function Distributions.kldivergence(p::LogNormal, q::LogNormal)
```

Comment:
The `Distributions.` is not needed inside of the same module

Reply:
Thanks! I hastily copied this from my drafts in a different environment.